linuxkit/linuxkit
# sbom support

**- What I did**

Add support for:

* using buildkit's SBoM scanner when running `lkt pkg build`, so that they are on the OCI image
* consolidating SBoMs from individual OCI images into a single one when running `lkt build`

Of course with CLI flags to enable/disable/control, and documentation.

While I was at it:

* fixed a bug in `dockerRun()`
* fixed some minor linting issues
* upgraded the default buildkit builder container version from `v0.11.0-rc2` to `v0.12.3`

**- How I did it**

Wrote documents, changed code.

**- How to verify it**

If CI passes, it works. I tested lots of different scenarios as well.

Test added to ensure the SBoM is in place

**- Description for the changelog**


Support for SBoMs in packages and in final image version
